fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1026,6 +1026,15 @@ class RYWImpl {
 		return Void();
 	}
 
+	ACTOR static void simulateTimeoutInFlightCommit(ReadYourWritesTransaction* ryw_) {
+		state Reference<ReadYourWritesTransaction> ryw = Reference<ReadYourWritesTransaction>::addRef(ryw_);
+		ASSERT(ryw->options.timeoutInSeconds > 0);
+		if (!ryw->resetPromise.isSet()) ryw->resetPromise.sendError(transaction_timed_out());
+		wait(delay(deterministicRandom()->random01() * 5));
```

Comment:
Would the delay time be useful for users to debug any potential bug this new PR helps to uncover? 
The delay time may be useful, "although unreliably", to reason about the commit ordering.